pkg/postage/batchservice/batchservice_test.go

```diff
@@ -47,6 +47,10 @@ type mockBatchListener struct {
 	diluteCount int
 }
 
+func (m *mockBatchListener) HandleStampExpiry(id []byte) {
```

Comment:
Check my other comment. This should be removed. We should not be changing the BatchEventListener interface